### ISSUE TITLE
Produce Documentation

### ISSUE THREAD

**Issue by pslobo** [careteditor/issues#411]
I'm all for adding new and exciting features and fixing bugs, but at this point in time, I honestly think you also need to spend some time creating decent documentation. There is a wealth of features that are hidden and users, both new and seasoned have much to gain from learning about those features.

Reading about all the features and how they work would also help a great deal for users undecided about purchasing the app. 😉 

I suppose you could host the documentation on GitHub and that way you could possibly get soem help in writing it.

**Comment by BsNoSi** [comment]
To invite everyone, I created a first [wiki page](https://github.com/careteditor/caret/wiki).

**Comment by pslobo** [comment]
Thanks @BsNoSi that was a great idea.

When I have a little free time I'll be sure to add to the wiki.

**Comment by EnCey** [comment]
Case in point: what is the "liquid markup" feature that was added in 3.0? 

Is it a subset of [Shopify's Liquid template language](https://shopify.github.io/liquid/)? I was very intrigued when I first read the release notes but found no information at all about it, and didn't figure anything out either by playing around in the editor (although `{% %}` tags seem to be recognized, but none I tried worked).
